remote/constants.py

```diff
@@ -0,0 +1,8 @@
+# coding=utf-8
```

Comment:
Better to move this constants into the camayoc.constants module, also I think adding a `VCENTER_` would be good to give more context and also namespace them

Reply:
Yes I'll address this when I re-organize.